src/Day.jsx

```diff
@@ -7,6 +7,14 @@ let Day = React.createClass({
 
   propTypes: TimeGrid.propTypes,
 
+  getDefaultProps() {
+    return {
+      now: new Date(),
+      type: 'gutter',
+      timeslots: 2
+    }
```

Comment:
Ahh right. I believe in that case it makes sense to not mark them as required (since they are actually provided), and provide the defaults at the lowest level they are needed. That shouldn't trigger warning right?

second choice for me would be 1, since it mirrors how we spread propsTypes